cluster-scope/base/core/namespaces/smart-village-view/resourcequota.yaml

```diff
@@ -0,0 +1,12 @@
+apiVersion: v1
+kind: ResourceQuota
+metadata:
+  name: opf-ci-pipelines-custom
```

Comment:
Nit, please change the name to match the namespace name.

```suggestion
  name: smart-village-view-custom
```

Reply:
Great feedback @tumido , I have renamed my resourcequota.